Client/Clique/Common/Models/UserProfile.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  UserProfile.swift
+//  Clique
+//
+//  Created by Infinum on 29.11.2022..
+//
+
+import Foundation
+
+struct UserProfile: Codable {
+    let user_id: Int
+    let name: String
+    let surname: String
+    let email: String
+    let contact_no: String
+    let birth_data: String
+    let profile_pic: String
+    let bio: String
+}
+
+struct UserProfileUpdateData: Codable {
+    let name: String
+    let surname: String
+    let email: String
+    let gender: String
+    let contact_no: String
+    let birth_data: String
+    let profile_pic: String
+    let bio: String
```

Comment:
Why cannot this be User?

Reply:
Ne znam je li mogu bit (tj. ne znam kako da to napravim) pod jednim jer mi UserProfile sluzi za dohvacanje podataka o Useru i njime punim profil, a UserProfileUpdateData mi sluzi za spremanje podataka kad se updatea user i taj model ne sadrzi id jer mi nije potrebno ga slati.